dotnet-authserver/tests/TeacherIdentity.AuthServer.Tests/EndpointTests/SignIn/Trn/AwardedQtsPageTests.cs

```diff
@@ -129,5 +129,9 @@ public async Task Post_ValidForm_SetsAwardedQtsOnAuthenticationStateRedirectsToC
         {
```

Comment:
We could also add checks to other tests for filling in default values if required